packages/sku/src/services/vite/helpers/config/dependencyGraph.ts

```diff
@@ -0,0 +1,138 @@
+import assert from 'node:assert';
+import path from 'node:path';
+import resolveSync from 'resolve-from';
+
+import fse from 'fs-extra';
```

Comment:
I know that crackle used it, but I'd prefer not to take a dependency on `fs-extra` just for the `readJson` function. It's easy to implement with native `node:fs`.